target/luaunit-reports/test_errormsg.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is it intended to commit these files?

Reply:
Added to .gitignore (was autogenerated by GitHub).

Reply:
But it's still there....

Reply:
I think you have to remove it if it was added to git once